### DISCUSSION TITLE
Problem with Installing Packages

### DISCUSSION THREAD

**Discussion by Nielssss99** [ci-group/revolve2#442] [Ask for help]
I am encountering the following error when running the sh student_install.sh command required to load the packages into the virtual environment. My laptop has Windows 11.

c_gpio.c
  source/c_gpio.c(27): fatal error C1083: Cannot open include file: 'sys/mman.h': No such file or directory
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.36.32532\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
  [end of output]
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for RPi.GPIO
  Running setup.py clean for RPi.GPIO
  Building wheel for spidev (pyproject.toml): started
  Building wheel for spidev (pyproject.toml): finished with status 'error'
  error: subprocess-exited-with-error
  Building wheel for spidev (pyproject.toml) did not run successfully.
  exit code: 1
  [26 lines of output]
  C:\Users\niels\AppData\Local\Temp\pip-build-env-l8i17oca\overlay\Lib\site-packages\setuptools\dist.py:472: SetuptoolsDeprecationWarning: Invalid dash-separated options
  !!

I already updated pip and setuptools, as proposed in an existing discussion. How can I tackle this problem?

Niels

**Comment by Nielssss99** [comment]
Issue solved. For people with similar problems in the future: pip installling "-e modular_robot_physical[remote]" by the use of WSL works. Installing "revolve2.simulators.mujoco_simulator" with WSL did not work, command prompt of Windows did work for this module.

**Reply by oliverweissl** [reply]
Im happy to hear that you solved the issue. Do you still have the error that you got when installing the mujoco_simulator on WSL?

**Reply by Nielssss99** [reply]
No. I was not able to collect all modules by either using WSL or Windows. So, I used both terminals depending on the module. For the mujoco_simulator WSL did not work, but Windows did. For the modular_robot_physical this was the other way around.
